easybuild/easyconfigs/b/Biggus/Biggus-0.13.0-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'Biggus'
+version = '0.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/SciTools/biggus/'
+description = """Virtual large arrays and lazy evaluation."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/SciTools/Biggus/archive/']
```

Comment:
Not really.  I thought they wrote in their docs that one should pip
install from Github, but now I cannot find that ...  If you prefer, I
can change to PyPI.